controller/skillreviewscontroller.go

```diff
@@ -83,6 +87,45 @@ func (c *SkillReviewsController) getAllSkillReviews() error {
 	return err
 }
 
+func (c *SkillReviewsController) getSkillReviewForSkill(skill_id string) error {
```

Comment:
The name of this method confused me initially. I think changing the name to `getReviewsForSkill` (or `getSkillReviewsForSkill) would be more clear.